addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -82,7 +82,7 @@
             <field name="refuse_reason_id" invisible="1"/>
             <header>
                 <button string="Create Employee" name="create_employee_from_applicant" type="object" data-hotkey="v" groups="hr_recruitment.group_hr_recruitment_user"
-                        class="o_create_employee" attrs="{'invisible': ['|',('emp_id', '!=', False),('active', '=', False)]}"/>
+                        class="o_create_employee" attrs="{'invisible': ['|', '|', ('emp_id', '!=', False), ('active', '=', False), ('date_closed', '=', False)]}"/>
```

Comment:
rather `('application_status', '!=', 'hired')`

Honestly, `('date_closed', '=', False)` and  `('application_status', '!=', 'hired') `checks the same thing, but IMO later one is more intuitive. You read and you know what's up, for `date_closed` I had to check in the code how it is computed. 

@jugj-odoo up to you to change or leave :smile: